### PR TITLE
'Projects' and 'About' sections add

### PR DESCRIPTION
**Pull request by Kambuchi**
_**Projects section**_

-Section with class "projects" created
-Add box structure for each product:

>Demonstrative image
>Project title
>Built with
>Button to see the full content

-Update fonts sizes

-Added images files

_**About section**_

-Section with class "about" created

>Added h2 as title

>Added a paragraph

>Added a button:
>>Text (as span)
> >Download icon (as img)

-Div with class "knowledge" created

>Added h3 as title
>Added ul as list of knowns
>Added background image w/ css

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -28,6 +28,68 @@ <h1>Hector<br>Pereira</h1>
...
+        <section class="about">
+            <h2>About</h2>
+            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, 
+            sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
+            Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris 
+            nisi ut aliquip ea eiusmodz</p>
+            <button>
+                <span>Get my CV</span>
+                <img src="img/download.png" alt="download" width="25">
+            </button>
```
Comment (mricanho) [inline index.html:91]: - [ ] You missed the technologies categories, please add it.

image

**Review by mricanho — CHANGES_REQUESTED** [review]
Hi @Kambuchi, 

Good job so far! 🔥🔥 This was an excellent first review, you use very well the semantic HTML, your gitflow and README file are correct.

There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️

- [ ] From now on, redact your commit messages in present-tense and imperative style,[ check this link](https://stackoverflow.com/questions/3580013/should-i-use-past-or-present-tense-in-git-commit-messages) for more info about how this is a good practice

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

missed by

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (descholar-ceo) [reply index.html:91]: :+1: Issue addressed successfully :heavy_check_mark:

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
Hi <STUDENT_MENTION>, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
- [ ] As pointed out by the previous reviewer, you should edit your commit messages which are in past tense and make them present or imperative mode, kindly edit this commit message: `Added boxes to projects` and make it a present mode. 

Tip: You can learn how to edit your commit messages using [this video](https://www.loom.com/share/00771f369b7b47fc968c019ecf0d3ba6)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Kambuchi** [comment]
I understand now, I'll check how to edit my commits.

**Review by henatan99 — APPROVED** [review]
Hi Kambuchi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

- [ ] OPTIONAL: I want to leave you an optional comment here. I know you did not change the previous commits as the TSE pointed out. I understand that changing past commits requires a bit of a workaround and I also believe you took a lesson on how to write a good commit message. Please, also take a lesson about `git rebase`, which lets you make changes on past commit messages. You can watch [this video](https://www.loom.com/share/00771f369b7b47fc968c019ecf0d3ba6) 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**